File: src/polaplib/scripts/pretend_work.py
#!/usr/bin/env python3
# Version: v0.1.0
import sys, time

if len(sys.argv) < 3:
    print("Usage: pretend_work.py <seconds> <outfile>", file=sys.stderr)
    sys.exit(2)
sec = float(sys.argv[1])
out = sys.argv[2]
time.sleep(sec)
with open(out, "w") as f:
    f.write("ok\n")
print(f"wrote {out}")

File: src/polaplib/scripts/pretend_work.R
#!/usr/bin/env Rscript
# Version: v0.1.0
args <- commandArgs(trailingOnly = TRUE)
if (length(args) != 2) {
  write("Usage: plot_hist.R <lengths.tsv> <out.pdf>", stderr())
  quit(status = 2)
}
x <- read.table(args[1], sep = "\t", header = FALSE, col.names = c("id", "len"))
pdf(args[2], width = 6, height = 4)
hist(x$len, main = "Lengths", xlab = "bp")
dev.off()
